examples/csi-storageclass-linear-xfs.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: csi-lvm-sc-linear-xfs
+provisioner: lvm.csi.metal-stack.io
+reclaimPolicy: Delete
+volumeBindingMode: WaitForFirstConsumer
+allowVolumeExpansion: true
+parameters:
+  type: "linear"
+  fsType: xfs
```

Comment:
some sort of validation is required what fsType(s) are supported, imagine someone specifies btrfs here

Reply:
Firstly, fsType is csi field: https://pkg.go.dev/github.com/container-storage-interface/spec/lib/go/csi#VolumeCapability_MountVolume

Secondary, btrfs will work also if the driver will be installed in the container.
Range of the values is limited by fs drivers in the container.

Reply:
In other csi drivers (for example https://github.com/vmware/cloud-director-named-disk-csi-driver.git) I don't see any validation of fsType.

Reply:
This might be the case, but it would be great if the consumer of csi-driver-lvm will see a proper error message that the given fsType is not supported.

This can be done for example with https://pkg.go.dev/os/exec#LookPath to `mkfs.fsType`

Reply:
And when the validation must be occured? Via ValidatingWebhookConfiguration? On provisioning? On attaching?

Reply:
At least the error should be so informative that the user can get an idea why his PV is not provisioned. If i remember correctly, errors raised even from the nodeserver are shown in `kubectl describe pv,pvc`. If this is achieved i am fine.

Reply:
Now I have tested the current code. And I see in pod's events:

`MountVolume.SetUp failed for volume "pvc-f1793398-f0ba-440b-bf9b-da55f7f1bc8d" : rpc error: code = Unknown desc = unable to mount lv: unable to format lv:pvc-f1793398-f0ba-440b-bf9b-da55f7f1bc8d err:exec: "mkfs.btrfs": executable file not found in $PATH output:`

Isn't it enough?

Reply:
Sure.